Bottom: 145c31a6954c193a2ed1d3716d0fb4db0b11cd14
Top:    0532a4014e54d2df96b1b5cbd60e1bca852de751
Author: Dotan Barak <dotanb@dev.mellanox.co.il>
Date:   2012-10-09 12:27:49 +0000

[2/5,librdmacm] rsocket: added missing break

Signed-off-by: Dotan Barak <dotanb@dev.mellanox.co.il>
Reviewed-by: Yishai Hadas <yishaih@mellanox.com>
Signed-off-by: Sean  Hefty <sean.hefty@intel.com>


---

diff --git a/src/rsocket.c b/src/rsocket.c
index 5b65b12..cc5effe 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1859,6 +1859,7 @@ int rsetsockopt(int socket, int level, int optname,
 		default:
 			break;
 		}
+		break;
 	case SOL_RDMA:
 		if (rs->state >= rs_opening) {
 			ret = ERR(EINVAL);
